src/Http/GarminClient/GarminAuthenticator.php

```diff
@@ -43,6 +46,10 @@ class GarminAuthenticator
 
     protected string $consumerSecret = '';
 
+    protected ?CodeProviderInterface $mfaCodeProvider = null;
```

Comment:
This was a bit tricky/weird.  Ideally, this would be injected from the container, but I wanted to use the `askHidden` feature on `StyleInterface`.  This is created at runtime after the container is built, so I added a setter for it and made it nullable.  It can then be set once the style is created. If it isn't set when MFA is requested, an error will be thrown.